### PR TITLE
Update avro & antlr

### PR DESCRIPTION
**Pull request by wendigo**
## Release notes

( ) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Review by jklamer — APPROVED** [review]
Is this change only need for ensuring JSON pretty is used in printing?

**Comment by wendigo** [comment]
@jklamer `toString` methods are deprecated and we treat warnings as error in this module :) Hence the change (the approach is called out in the javadocs)

File: plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/confluent/AvroSchemaConverter.java [hunk 1]
```diff
@@ -58,6 +59,8 @@
 
 public class AvroSchemaConverter
 {
+    private static final SchemaFormatter JSON_PRETTY_FORMATTER = SchemaFormatter.getInstance("json/pretty");
```
Comment (ebyhr) [inline plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/confluent/AvroSchemaConverter.java:62]: Why not use `new JsonSchemaFormatter(true)`?

**Review by ebyhr — APPROVED** [review]

File: plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/confluent/AvroSchemaConverter.java (hunk 1 above)
Comment (wendigo) [reply plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/confluent/AvroSchemaConverter.java:62]: The factory is mentioned jn the docs so I wanted to use most "canonical" way

Comment (wendigo) [reply plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/confluent/AvroSchemaConverter.java:62]: Screenshot 2024-08-06 at 09 21 58

**Pull request merged**